roles/lib_openshift/src/ansible/oc_atomic_container.py

```diff
@@ -9,7 +9,9 @@
 def _install(module, container, image, values_list):
     ''' install a container using atomic CLI.  values_list is the list of --set arguments.
     container is the name given to the container.  image is the image to use for the installation. '''
-    args = ['atomic', 'install', "--system", '--name=%s' % container] + values_list + [image]
+    # NOTE: system-package=no is hardcoded. This should be changed to an option in the future.
+    args = ['atomic', 'install', '--system', '--system-package=no',
```

Comment:
@ganhuang this another location where we are hardcoding in the ``--system-package=no``.